_build/data/transport.core.system_settings.php

```diff
@@ -82,6 +82,15 @@
   'area' => 'system',
   'editedon' => null,
 ], '', true, true);
+$settings['auto_detect_context']= $xpdo->newObject(modSystemSetting::class);
+$settings['auto_detect_context']->fromArray([
+  'key' => 'auto_detect_context',
+  'value' => true,
```

Comment:
It has to default to false.

Reply:
Ok, thank you

Reply:
@Jako Hi, can you see this code and mark change request as completed?